tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "module": "es2015",
+    "moduleResolution": "node",
+    "lib": ["es2017", "dom", "dom.iterable"],
+    "declaration": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "strictFunctionTypes": true,
```

Comment:
I'd recommend `"strict": true`

Reply:
I think I had trouble with the implicit any's from importing the MDC Foundations, but I'll give it a shot.

Reply:
I can turn `strict: true` on, but I also have to disable `noImplicitUseStrict`.

Leaving `noImplicitAny` on, otherwise I'd have to provide typings for all the Foundations.